### ISSUE TITLE
Add a PR template

### ISSUE THREAD

**Issue by MadhavBahlMD** [pyista/pythonista#9]
**Is your feature request related to a problem? Please describe.**
No, it is not related to a problem.

**Describe the solution you'd like**
It would be great if there would be a PR template regarding the checklist of the actions performed and the description

**Comment by ajmalsiddiqui** [comment]
You're right, we probably need a PR template. Apart from the usual components of the PR template, anything specific to Pythonista should also be added. As of now, I can't think of anything specific off the top of my head, but the discussion can happen in this issue thread.

This issue is up for grabs!

**Comment by phyyyl** [comment]
What special pythonista components are these?

**Comment by 1cedsoda** [comment]
The Hacktoberfest is over.

**Comment by gshanbhag525** [comment]
Can I add a PR template for this repo?
